spec/models/game_spec.rb

```diff
@@ -1,6 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe Game, type: :model do
+  describe '#check?' do
+    it 'returns true if king is in check' do
+      white_player = FactoryGirl.create(:user)
+      black_player = FactoryGirl.create(:user)
```

Comment:
Since these are used multiple times, they can be extracted to Let's

http://www.betterspecs.org/#let